src/Features/Core/Portable/ConflictMarkerResolution/AbstractConflictMarkerCodeFixProvider.cs

```diff
@@ -45,91 +45,163 @@ public override async Task RegisterCodeFixesAsync(CodeFixContext context)
+            var lines = text.Lines;
+            for (var i = startLine.LineNumber + 1; i < lines.Count; i++)
+            {
+                var currentLine = lines[i];
+                if (IsConflictMarker(currentLine, ch))
+                {
+                    foundLine = currentLine;
+                    return true;
+                }
+            }
+
+            foundLine = default;
+            return false;
+        }
 
-            while (true)
+        private static bool TryFindLineBackwards(TextLine startLine, char ch, out TextLine foundLine)
+        {
+            var text = startLine.Text!;
+            var lines = text.Lines;
+            for (var i = startLine.LineNumber - 1; i >= 0; i--)
             {
-                var index = GetEqualsConflictMarkerIndex(text, token, afterPosition: startTrivia.SpanStart);
-                if (index >= 0)
+                var currentLine = lines[i];
+                if (IsConflictMarker(currentLine, ch))
                 {
-                    var leadingTrivia = token.LeadingTrivia;
-
-                    if (index + 3 < token.LeadingTrivia.Count)
-                    {
-                        // normal case where there us =====, then dead code, then >>>>>>
-
-                        var equalsTrivia = leadingTrivia[index];
-                        var endOfLineTrivia = leadingTrivia[index + 1];
-                        var disabledTrivia = leadingTrivia[index + 2];
-                        var endTrivia = leadingTrivia[index + 3];
-
-                        if (_syntaxKinds.EndOfLineTrivia == endOfLineTrivia.RawKind &&
-                            _syntaxKinds.DisabledTextTrivia == disabledTrivia.RawKind &&
-                            IsConflictMarker(text, endTrivia, '>'))
-                        {
-                            return (equalsTrivia, endTrivia);
-                        }
-                    }
-
-                    if (index + 2 < token.LeadingTrivia.Count)
-                    {
-                        // case where there is ===== followed by >>>>>>  on the next line.
-
-                        var equalsTrivia = leadingTrivia[index];
-                        var endOfLineTrivia = leadingTrivia[index + 1];
-                        var endTrivia = leadingTrivia[index + 2];
-
-                        if (_syntaxKinds.EndOfLineTrivia == endOfLineTrivia.RawKind &&
-                            IsConflictMarker(text, endTrivia, '>'))
-                        {
-                            return (equalsTrivia, endTrivia);
-                        }
-                    }
+                    foundLine = currentLine;
+                    return true;
                 }
+            }
 
-                token = token.GetNextToken(includeZeroWidth: true);
-                if (token.RawKind == 0)
+            foundLine = default;
+            return false;
+        }
+
+        private static bool IsConflictMarker(TextLine currentLine, char ch)
```

Comment:
So this means we don't handle this properly, right? I think it's not unreasonable, just want to confirm

![image](https://user-images.githubusercontent.com/788783/77128012-d5a89080-6a0b-11ea-8f75-447f4dcbe1c6.png)